### DISCUSSION TITLE
1.9.0

### DISCUSSION THREAD

**Discussion by AndreyPavlenko** [AndreyPavlenko/Fermata#205] [Announcements]
Felex addon - a simple language tutor.
Xposed hook to enable the app in AA.
Libraries update.

<hr /><em>This discussion was created from the release <a href='https://github.com/AndreyPavlenko/Fermata/releases/tag/'>1.9.0</a>.</em>

**Comment by luxysiv** [comment]
Darkmode on YouTube not work, please fix it

**Reply by majstermsi** [reply]
Hi , could you compare code with 1.8.10 version to eliminate White screen in 1.9 version and 1.8.11 ? 1.8.10 is perfect however lack of auto resume is frustrating.

**Reply by luxysiv** [reply]
It's fine on newest Auto version. Perfect

**Comment by RoomMT** [comment]
from v1.8.11, when play youtube and swich other app it stop play. and come back fermata screen video and menu control not work.

**Reply by majstermsi** [reply]
It is the same issue I described however in my device is also White screen.

**Comment by luxysiv** [comment]
On newest Auto version, Fermata works perfectly

**Reply by gaduaxe** [reply]
> On newest Auto version, Fermata works perfectly

Hi
How did you do? With root or non root phone?

I cannot use on AA with Car, only successful with 1st time, 2nd time was not

Thank you

**Reply by luxysiv** [reply]
> > On newest Auto version, Fermata works perfectly
> 
> Hi How did you do? With root or non root phone?
> 
> I cannot use on AA with Car, only successful with 1st time, 2nd time was not
> 
> Thank you

I used on Android 13 no root. Try install Fermata by King Installer

**Reply by majstermsi** [reply]
Even on cool walk White screen and GUI destroy appears. I will try AA 8.8

**Reply by majstermsi** [reply]
Still corrupted new 8.9 beta

**Reply by majstermsi** [reply]
Still corrupted new 8.9 beta

**Comment by majstermsi** [comment]
![Uploading 20230210_152545.jpg…]()

**Comment by manfrommars7** [comment]
> Felex addon - a simple language tutor. Xposed hook to enable the app in AA. Libraries update.
> 
> _This discussion was created from the release [1.9.0](https://github.com/AndreyPavlenko/Fermata/releases/tag/)._

Please add gps navigation bar on the top while using youtube.

**Reply by AndreyPavlenko** [reply]
What do you mean by "gps navigation bar"?

**Reply by manfrommars7** [reply]
Like the one before coolwalk.

**Reply by manfrommars7** [reply]
But before it was on bottom of screen.

**Comment by manfrommars7** [comment]
I am using latest android auto coolwalk.
